api/envoy/extensions/filters/http/ext_proc/v3/processing_mode.proto

```diff
@@ -68,6 +69,31 @@ message ProcessingMode {
     // chunk. If the body exceeds the configured buffer limit, then the body contents
     // up to the buffer limit will be sent.
     BUFFERED_PARTIAL = 3;
+
+    // [#not-implemented-hide:]
+    // Envoy streams the body to the server in pieces as they arrive. The data is not stored in
+    // the internal queue. Only data size is stored there for tracking and flow control purpose.
+    //
+    // Server may choose to buffer any number chunks of data before processing them.
+    // After finish buffering, server processes the buffered data. Then it splits the processed
+    // data into any number of chunks, and stream them back to Envoy one by one.
```

Comment:
This can be addressed later. The ext_proc developer needs to know the maximum size of thew chunk it is allowed to send. We should either recommend a safe default size or send the maximum size Envoy can accept in the ext_proc request.

Reply:
I recall @tyxia added a configurable max chunk limit in gRPC to avoid the response causing OOM in Envoy.  And  the default max limit is 128KB.  Please correct me if I am wrong @tyxia .

BTW, as the approach completely removed the input-queue based flow control. The solution has dependency on @tyxia's 4-way flow control to avoid OOM.

Reply:
This is not the same. The extension developer needs to know how much data can be sent safely.

Reply:
Added below text:

// The individual chunk size is recommended to be smaller than 64K bytes, or smaller than
// :ref:`max_receive_message_length <envoy_v3_api_field_config.core.v3.GrpcService.EnvoyGrpc.max_receive_message_length>`
// if EnvoyGrpc is used, and ``max_receive_message_length`` is configured.